src/applications/mhv/medications/containers/LandingPage.jsx

```diff
@@ -98,11 +98,11 @@ const LandingPage = () => {
           <section>
             <h1
               data-testid="landing-page-heading"
-              className="vads-u-margin-top--4 vads-u-margin-bottom--0"
+              className="medium-screen:vads-u-margin-top--4 medium-screen:vads-u-margin-bottom--0 vads-u-margin-top--3 vads-u-margin-bottom--1"
```

Comment:
I was using this recently and was told to make sure we use small-screen for mobile

Reply:
good catch ill do that as well 👍